src/ert/plugins/hook_implementations/help_resources.py

```diff
@@ -1,6 +1,6 @@
 import ert
 
 
-@ert.plugin(name="ert")  # type: ignore
-def help_links():
+@ert.plugin(name="ert")
+def help_links():  # type: ignore
```

Comment:
Is it possible to avoid the ` # type: ignore` by adding `-> dict[str, str]` instead?